### ISSUE TITLE
  NestedScopeFunctions not triggered

### ISSUE THREAD

**Issue by Dosexe** [detekt/detekt#5355]
## Expected Behavior
NestedScopeFunctions should be triggered for
```kotlin
        "text".let {
            "".let {
                ""
            }
        }
```

## Observed Behavior
Rule is not triggered as expected

## Steps to Reproduce
  ```yaml
NestedScopeFunctions:
    active: true
    threshold: 1
    functions:
      - 'kotlin.apply'
      - 'kotlin.run'
      - 'kotlin.with'
      - 'kotlin.let'
      - 'kotlin.also'
      - 'let'
```
just trigger for code above
## Your Environment

* Version of detekt used: 1.21.1
* Version of Gradle used (if applicable): 7.4.2
* Gradle scan link (add `--scan` option when running the gradle task):
* Operating System and version:
* Link to your project (if it's a public repository):

**Comment by schalkms** [comment]
I couldn't reproduce this issue. Since this rule uses type resolution, I suppose you didn't enable it. See [here](https://detekt.dev/docs/gettingstarted/type-resolution) for more details.
